cpp/include/cugraph_c/layout_algorithms.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cugraph_c/error.h>
+#include <cugraph_c/graph.h>
+#include <cugraph_c/graph_functions.h>
+#include <cugraph_c/random.h>
+#include <cugraph_c/resource_handle.h>
+
+/** @defgroup layout Layout algorithms
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief   Force Atlas 2
+ *
+ * NOTE: This currently wraps the legacy force atlas 2 implementation and is only
+ * available in Single GPU implementation.
+ *
+ * @param [in]   handle          Handle for accessing resources
+ * @param [in]   graph           Pointer to graph.  NOTE: Graph might be modified if the storage
+ *                               needs to be transposed
+ * @param [out]  pos             Position of the vertices
```

Comment:
Note that in the C++ code, pos is a 2-d array and assumes that the output is ordered by internal vertex ids.

Generally, the C API hides the renumbering of vertices from the user (we automatically renumber/unrenumber in the C API before calling the C++ API.  As such, this really needs include the vertex ids.

I'd suggest creating a `cugraph_layout_result_t` type and including vertex id, x position, y position so that you can easily create a data frame for the resulting python.